src/main/java/cz/tstrecha/timetracker/controller/v1/TaskController.java

```diff
@@ -40,4 +38,11 @@ public ResponseEntity<TaskDTO> createEmptyTask(@PathVariable IdentifierType iden
                                                    @InjectLoggedUser LoggedUser user){
         return new ResponseEntity<>(taskService.createEmptyTask(identifier,identifierValue,user), HttpStatus.CREATED);
     }
+
+    @GetMapping("/search/{query}?limit=")
+    public ResponseEntity<List<TaskDTO>> searchTasks(@RequestParam(defaultValue = "5", required = false) Long limit,
```

Comment:
best practise is to have it in this order

- path variables
- request params
- request body
- injected stuff

In other words: just have query first and limit second